### PR TITLE
Add support for optarch=GENERIC.

### PR DESCRIPTION
**Pull request by terjekv**
Now against the correct branch...

### CODE REVIEW

File: EESSI-pilot-2020.09.sh [hunk 1]
```diff
@@ -31,6 +31,17 @@ export ARCH=$(uname -m)
 export EESSI_PREFIX=${CVMFS_REPO}/${EESSI_PILOT_VERSION}
 export EPREFIX=${EESSI_PREFIX}/compat/${ARCH}
 
+DETECTION_PARAMETERS=''
+GENERIC=0
+EB='echo eb'
```
Comment (boegel) [inline EESSI-pilot-2020.09.sh]: strip out the `echo`?

File: EESSI-pilot-2020.09.sh [hunk 2]
```diff
@@ -31,6 +31,17 @@ export ARCH=$(uname -m)
...
+if [[ "$1" == "--generic" || "$EASYBUILD_OPTARCH" == "GENERIC" ]]; then
+    echo_yellow ">> GENERIC build requested, taking appropriate measures!"
+    DETECTION_PARAMETERS="$DESTINATION_PARAMETERS --generic"
+    GENERIC=1
+    EB='echo eb --optarch=GENERIC'
```
Comment (boegel) [inline EESSI-pilot-2020.09.sh]: `echo` should also be removed here?

File: EESSI-pilot-2020.09.sh [hunk 3]
```diff
@@ -31,6 +31,17 @@ export ARCH=$(uname -m)
 export EESSI_PREFIX=${CVMFS_REPO}/${EESSI_PILOT_VERSION}
 export EPREFIX=${EESSI_PREFIX}/compat/${ARCH}
 
+DETECTION_PARAMETERS=''
+GENERIC=0
+EB='echo eb'
+if [[ "$1" == "--generic" || "$EASYBUILD_OPTARCH" == "GENERIC" ]]; then
+    echo_yellow ">> GENERIC build requested, taking appropriate measures!"
+    DETECTION_PARAMETERS="$DESTINATION_PARAMETERS --generic"
```
Comment (boegel) [inline EESSI-pilot-2020.09.sh]: `s/DESTINATION_PARAMETERS/DETECTION_PARAMETERS/`?

**Review by boegel — CHANGES_REQUESTED** [review]

File: EESSI-pilot-2020.09.sh (hunk 1 above)
Comment (terjekv) [reply EESSI-pilot-2020.09.sh]: ./shame

File: EESSI-pilot-2020.09.sh (hunk 2 above)
Comment (terjekv) [reply EESSI-pilot-2020.09.sh]: ./more_shame

File: EESSI-pilot-2020.09.sh (hunk 3 above)
Comment (terjekv) [reply EESSI-pilot-2020.09.sh]: This is what I get for using VIM on a remote host! :)

**Review by boegel — APPROVED** [review]

**Pull request merged**